python/paddle/v2/fluid/framework.py

```diff
@@ -565,6 +602,26 @@ def sync_with_cpp(self):
         for block in self.blocks:
             block.sync_with_cpp()
 
+    def copy_param_info_from(self, other):
+        """
+        Copy the information of parameters from other program. 
+        Args:
+            other(Program): Other program
+
+        Returns:
+            None
+        """
+        if not isinstance(other, Program):
+            raise TypeError("copy_param_info_from should be invoked with "
+                            "Program")
+
+        if len(self.blocks) != len(other.blocks):
+            raise ValueError("copy_param_info_from should be invoked with two "
+                             "program, with represent the same topology")
+
+        for dst_block, src_block in six.zip(self.blocks, other.blocks):
+            dst_block.copy_param_info_from(src_block)
```

Comment:
I think `Parameter` can only be in the root block?

Reply:
You're right.